pdc/apps/common/viewsets.py

```diff
@@ -200,7 +200,8 @@ def initial(self, request, *args, **kwargs):
             self._check_ordering_keys(request)
 
     def _check_ordering_keys(self, request):
-        ordering_keys = request.query_params.get('ordering')
+        # Allow ordering to be defined multiple times and also as comma separated list.
+        ordering_keys = ','.join(request.query_params.getlist('ordering'))
```

Comment:
To be more clear, calling `client.get_paged(client["release-variants"]._, ordering=["variant_uid", "release"])` will make the `ordering` be a list (this happens even when passing multiple same parameters in URI).

All these parameters are used when sorting in Django REST framework (they are passed to `OrderingFilter` base class for our `RelatedNestedOrderingFilter`).

BTW, both ways still work so you can use `?ordering=aaa,bbb&ordering=ccc`.

I think the `_check_ordering_keys()` only exists so it can report to client if there is any problem. Perhaps it can be removed altogether later.